src/components/stats/PullStats.jsx

```diff
@@ -28,7 +28,6 @@ class PullStats extends Component {
         <Query
           query={gql`
                     {
-
                         repository(owner: "${this.state.username}", name: "${
             this.state.repository
```

Comment:
Deconstruct this Also run auto formatter on this file

Reply:
need this for GQL, deconstructed at least